pages/app-layout/runtime-drawers-with-updates.page.tsx

```diff
@@ -0,0 +1,196 @@
+          },
+        ],
+        onDrawerChange: event => {
+          setActiveDrawerId(event.detail.activeDrawerId);
+        },
+      };
+
+  const onChangeShowBadge = (checked: boolean) => {
+    setShowBadge(checked);
+    awsuiPlugins.appLayout.updateDrawer({
+      id: 'security',
+      badge: checked,
+    });
+  };
+
+  const onChangeTurnOffResize = (checked: boolean) => {
+    setTurnOffResizable(checked);
+    awsuiPlugins.appLayout.updateDrawer({
+      id: 'security',
+      resizable: !checked,
+    });
+  };
+
+  return (
+    <AppLayout
+      ariaLabels={appLayoutLabels}
+      breadcrumbs={<Breadcrumbs />}
+      content={
+        <ContentLayout
+          disableOverlap={true}
+          header={
+            <SpaceBetween size="m">
+              <Header
+                variant="h1"
+                description="Sometimes you need custom drawers to get the job done."
+                info={
+                  <Link
+                    data-testid="info-link-header"
+                    variant="info"
+                    onFollow={() => {
+                      setHelpPathSlug('header');
+                      setIsToolsOpen(true);
+                    }}
+                  >
+                    Info
+                  </Link>
+                }
+              >
+                Testing Custom Drawers with update!
+              </Header>
+
+              <SpaceBetween size="xs">
+                <Toggle checked={hasTools} onChange={({ detail }) => setUrlParams({ hasTools: detail.checked })}>
+                  Use Tools
+                </Toggle>
+
+                <Toggle checked={hasDrawers} onChange={({ detail }) => setUrlParams({ hasDrawers: detail.checked })}>
+                  Use Drawers
+                </Toggle>
+
+                <SpaceBetween size="xs">
+                  <h3>Security Drawer Updates</h3>
+                  <Toggle checked={showBadge} onChange={({ detail }) => onChangeShowBadge(detail.checked)}>
+                    Show Badge
+                  </Toggle>
+                  <Toggle
+                    data-testid="turn-off-resize-toggle"
+                    checked={turnOffResizable}
+                    onChange={({ detail }) => onChangeTurnOffResize(detail.checked)}
+                  >
+                    Turn off Resize
+                  </Toggle>
+                </SpaceBetween>
+              </SpaceBetween>
+            </SpaceBetween>
+          }
+        >
+          <Header
+            info={
+              <Link
+                data-testid="info-link-content"
+                variant="info"
+                onFollow={() => {
+                  setHelpPathSlug('content');
+                  setIsToolsOpen(true);
+                }}
+              >
+                Info
+              </Link>
+            }
+          >
+            Content
+          </Header>
+          <Containers />
+        </ContentLayout>
+      }
+      splitPanel={
+        <SplitPanel
```

Comment:
Similar to the above, split panel is also not needed for this demo and can be removed to keep it simpler

Reply:
removed